src/components/Customers.tsx

```diff
@@ -39,9 +33,12 @@ export const Customers = () => (
         viewport={{ once: true }}
         transition={{ duration: 0.4, delay: 0.3 }}
       >
-        <div className="shorebird-block-big-title mb-16 px-8 text-center sm:px-24 md:px-48">
-          Thousands of developers trust Shorebird to deliver millions of patches
-          each month.
+        <div className="shorebird-block-big-title mb-8 px-8 text-center sm:px-24 md:px-48">
+          Trust by thousands of developers
+        </div>
+
+        <div className="shorebird-content-title mb-16 px-8 text-center sm:px-24 md:px-48">
+          to deliver millions of patches every day!
```

Comment:
I think this is probably its own sentence, rather than a continuation of the title.
```suggestion
Shorebird deliver millions of patches for thousands of developers every month.
```